src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,82 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int DEFAULT_RADIUS = 10;
+    private static final int MAX_ISOSCELES_TRAPEZOID_SIDE_A = 20;
+    private static final int MAX_ISOSCELES_TRAPEZOID_SIDE_B = 10;
+    private static final int MAX_ISOSCELES_TRAPEZOID_WEIGHT = 15;
+    private static final int MAX_RECTANGLE_WIDTH = 10;
+    private static final int MAX_RECTANGLE_HEIGHT = 10;
+    private static final int MAX_SQUARE_SIDE = 20;
+    private static final int MAX_RIGHT_TRIANGLE_FIRST_LEG = 20;
+    private static final int MAX_RIGHT_TRIANGLE_SECOND_LEG = 20;
+    private Figure randomFigure;
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private String color = colorSupplier.getRandomColor();

```

Comment:
let's make the variable `String color` local as we want to get new color every time we call  getRandomFigure() method